nvi-rest/src/main/java/no/sikt/nva/nvi/rest/upsert/UpdateNviCandidateStatusHandler.java

```diff
@@ -65,7 +73,7 @@ private void verifyRequesteeIsCorrectCustomer(URI institutionId, URI currentCust
 
     private DbApprovalStatus toStatus(NviStatusRequest input, DbUsername username) {
         return new DbApprovalStatus(input.institutionId(), mapStatus(input.status()),
-                                    username, username, Instant.now());
+                                    username, username, Instant.now(), input.reason());
     }
```

Comment:
What about using builder instead of constructor. I mean we already have builder for this object